### PR TITLE
feat: a/b testing for new survey on webview page

### PR DESCRIPTION
**Pull request by chagong**
feat: a/b test new survey on webview page

image
image
image
image

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10719505
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10719506

### CODE REVIEW

File: packages/vscode-extension/src/extension.ts [hunk 1]
```diff
@@ -35,6 +35,21 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(new ExtTelemetry.Reporter(context));
 
   await exp.initialize(context);
+  TreatmentVariableValue.isExpandCard = (await exp
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/extension.ts:38]: This logic has been removed in dev since the experiment is done

**Comment by jaeyonglee8687** [comment]
There will be a couple of Visual Design that needs to be aligned with the VSC design style, but for the basic functionality-wise, I think it's okay. We will find some time to improve the frontend part later.

**Comment by jaeyonglee8687** [comment]
One minor quick thing. Give some margin between the submit button and the input box. It's too close.

File: packages/vscode-extension/src/extension.ts (hunk 1 above)
Comment (chagong) [reply packages/vscode-extension/src/extension.ts:38]: got it.

**Comment by chagong** [comment]
> There will be a couple of Visual Design that needs to be aligned with the VSC design style, but for the basic functionality-wise, I think it's okay. We will find some time to improve the frontend part later.

Yeah, sure.

File: packages/vscode-extension/src/resources/Strings.json [hunk 2]
```diff
@@ -197,7 +197,8 @@
       "emptyData": "Empty zip file",
       "downloadSampleFail": "Failed to download sample app",
       "quickStartPageTitle": "Quick Start",
-      "samplePageTitle": "Samples"
+      "samplePageTitle": "Samples",
+      "surveyPageTitle": "TeamsFx Survey"
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/resources/Strings.json:201]: Is 'Teamsfx' something we want to expose to user?

Comment (chagong) [reply packages/vscode-extension/src/resources/Strings.json:201]: good catch

**Review by swatDong — APPROVED** [review]
The task handler trigger point looks good to me.

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Pull request merged**